src/components/Restoration/BusinessName.vue

```diff
@@ -0,0 +1,288 @@
+          </div>
+          <div class="company-info">
+            <span class="subtitle">Expiry Date: </span>
+            <span class="info-text">{{expiryDate || 'Unknown'}}</span>
+          </div>
+          <div class="company-info">
+            <span class="subtitle">Status: </span>
+            <span class="info-text text-capitalize">{{nrStatus}}</span>
+          </div>
+        </template>
+      </v-col>
+    </v-row>
+
+    <!-- Name Request Applicant -->
+    <v-row v-if="hasNewNr" no-gutters class="sub-section">
+      <v-col cols="3">
+        <label class="pr-4">Name Request Applicant</label>
+      </v-col>
+
+      <v-col cols="7">
+        <div class="name-request-applicant-info">
+          <span class="subtitle">Name: </span>
+          <span class="info-text">{{nrApplicant.fullName}}</span>
+        </div>
+        <div class="name-request-applicant-info">
+          <span class="subtitle">Address: </span>
+          <span class="info-text">{{nrApplicant.fullAddress}}</span>
+        </div>
+        <div class="name-request-applicant-info">
+          <span class="subtitle">Email: </span>
+          <span class="info-text">{{nrApplicant.emailAddress || 'N/A'}}</span>
+        </div>
+        <div class="name-request-applicant-info">
+          <span class="subtitle">Phone: </span>
+          <span class="info-text">{{phoneNumber || 'N/A'}}</span>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF, EntitySnapshotIF, FlagsCompanyInfoIF, NameRequestApplicantIF, NameRequestIF }
+  from '@/interfaces/'
+import { ContactPointIF } from '@bcrs-shared-components/interfaces/'
+import CorrectNameOptions from '@/components/Restoration/CorrectNameOptions.vue'
+import AddNameTranslation from '@/components/common/AddNameTranslation.vue'
+import ListNameTranslations from '@/components/common/ListNameTranslations.vue'
+import { CommonMixin, DateMixin, NameRequestMixin } from '@/mixins/'
+import { CoopTypes, NameChangeOptions } from '@/enums/'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module/'
+
+@Component({
+  components: {
+    CorrectNameOptions,
+    AddNameTranslation
+  },
+  mixins: [
+    CommonMixin,
+    DateMixin,
+    NameRequestMixin
+  ]
+})
+export default class BusinessName extends Vue {
+  // for template
+  readonly GetCorpFullDescription = GetCorpFullDescription
+
+  @Prop({ default: false }) getComponentValidate!: boolean
+
+  // Global getters
+  @Getter getNameRequest!: NameRequestIF
+  @Getter getBusinessNumber!: string
+  @Getter getNameRequest!: NameRequestIF
+  @Getter getEntityType!: CorpTypeCd
+
+  // Global actions
+  // @Action setEditingCompanyName!: ActionBindingIF
+  // @Action setValidComponent!: ActionBindingIF
+
+  // *** TODO
```

Comment:
In progress, but I want to commit incrementally.

Right now this doesn't break anything (since it's only used by restoration filings).